com.microsoft.mrtk.uxcore/Keyboard/NonNative/NonNativeKeyboard.cs

```diff
@@ -180,9 +181,28 @@ public string Text
         [SerializeField, Tooltip("References the @ bottom panel.")]
         public GameObject emailBottomKeysSection = null;
 
+        /// <summary>
+        /// Used for changing the color of the icon to indicate if recording is active.
+        /// </summary>
+        [SerializeField, Tooltip("Used for changing the color of the icon to indicate if recording is active.")]
+        public Image dictationRecordIcon = null;
```

Comment:
Can you please look into hiding the microphone if there is no active dictation service.  Ideally, we would hide the microphone button if there is no dictation service.

Reply:
Hiding the microphone image if there is no DictationSubsystem